python/mxnet/executor.py

```diff
@@ -311,212 +309,3 @@ def debug_str(self):
             self.handle, ctypes.byref(debug_str)))
         return py_str(debug_str.value)
 
-def _split_input_slice(batch_size, work_load_list):
```

Comment:
why remove all this? Some user code may depend on these protected functions. Moving them can cause confusion